ipamutils/utils.go

```diff
@@ -34,19 +33,47 @@ type NetworkToSplit struct {
 	Size int    `json:"size"`
 }
 
-// InitNetworks initializes the broad network pool and the granular network pool
-func InitNetworks(defaultAddressPool []*NetworkToSplit) {
-	initNetworksOnce.Do(func() {
-		// error ingnored should never fail
-		PredefinedGranularNetworks, _ = splitNetworks(defaultGranularNetwork)
-		if defaultAddressPool == nil {
-			defaultAddressPool = defaultBroadNetwork
-		}
-		var err error
-		if PredefinedBroadNetworks, err = splitNetworks(defaultAddressPool); err != nil {
-			logrus.WithError(err).Error("InitAddressPools failed to initialize the default address pool")
-		}
-	})
+func init() {
+	var err error
+	if PredefinedGlobalScopeDefaultNetworks, err = splitNetworks(globalScopeDefaultNetworks); err != nil {
+		logrus.WithError(err).Error("InitAddressPools failed to initialize the global scope default address pool")
+	}
```

Comment:
Since this default initialization only occurs on compile-time constant input it must always work.  Therefore, it should be OK to panic() if some kind of error occurs.  This is similar to https://golang.org/pkg/regexp/#MustCompile which panics if a regex is malformed.  (again, one invokes this on compile-time regex strings that are known to be good.)